src/main/java/gwt/material/design/addins/client/fileuploader/MaterialFileUploader.java

```diff
@@ -511,8 +523,25 @@ public void onError(ErrorEvent<UploadFile> event) {
     }
 
     @Override
-    public void fireErrorEvent(String fileName, String lastModified, String size, String type, String responseCode, String responseMessage) {
-        ErrorEvent.fire(this, new UploadFile(fileName, new Date(lastModified), Double.parseDouble(size), type), new UploadResponse(responseCode, responseMessage));
+    public void fireErrorEvent(String fileName, String lastModified, String size, String type, String responseCode, String responseMessage, String responseBody) {
+        ErrorEvent.fire(this, new UploadFile(fileName, new Date(lastModified), Double.parseDouble(size), type), new UploadResponse(responseCode, responseMessage, responseBody));
+    }
+
+    @Override
+    public HandlerRegistration addUnauthorizedHandler(final UnauthorizedEvent.UnauthorizedHandler<UploadFile> handler) {
+        return addHandler(new UnauthorizedEvent.UnauthorizedHandler<UploadFile>() {
+            @Override
+            public void onUnauthorized(UnauthorizedEvent<UploadFile> event) {
+                if (isEnabled()) {
```

Comment:
Remove this enable check, since it isn't conditional to the widgets enabled state.